$OpenBSD: patch-src_editor_c,v 1.2 2019/01/21 20:13:44 landry Exp $

/usr/bin/ld: error: undefined symbol: search_for_keyword_fastly
>>> referenced by editor.c
>>>               beaver-editor.o:(markup_syhi_range)
/usr/bin/ld: error: undefined symbol: refresh_markers
>>> referenced by editor.c
>>>               beaver-editor.o:(markup_syhi_range)

Index: src/editor.c
--- src/editor.c.orig
+++ src/editor.c
@@ -537,7 +537,7 @@ inline gint is_matching_keyword (gint Lg, const gchar 
 }
 
 
-inline void refresh_markers (GtkTextBuffer * Buffer)
+void refresh_markers (GtkTextBuffer * Buffer)
 {
 	gint CurrentPage;
 	gint Lg;
@@ -894,7 +894,7 @@ void search_for_keyword_correctly (gint page, GtkTextB
 END_FCN}
 
 
-inline void search_for_keyword_fastly (gint page, GtkTextBuffer * Buffer,
+void search_for_keyword_fastly (gint page, GtkTextBuffer * Buffer,
 									   gchar * text, gint * i, gint offset,
 									   gint Lg)
 {
